example/cma_es_example.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Nick Thompson, 2024
+ * Use, modification and distribution are subject to the
+ * Boost Software License, Version 1.0. (See accompanying file
+ * LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+ */
+#if __APPLE__ || __linux__
+#include <signal.h>
```

Comment:
This should be every POSIX system right? You could `#if __has_include(<unistd.h>)` which is a requirement of every POSIX system.